readme.md

```diff
@@ -230,6 +266,12 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
-```
+db.instructors.update(
+  {"name" : "Andy"},
+  {$set {"wishlist_items":
+    {"description" : "Resin Laying Deer Figurine, Gold"}
+  }
+})
```

Comment:
Take a look at the [solution](https://github.com/ga-dc/wdi7_quiz10/tree/solution#question-14) for this one.